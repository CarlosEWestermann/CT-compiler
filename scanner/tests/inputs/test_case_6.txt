(int x, bool flag) >= int! complexfunction {
    int result;
    if (x < 10) {
        result = x * 2;
    } else {
        result = x / 2;
    };
    while (result < 100) {
        result = 10;
    };
    if (flag) {
        return result;
    } else {
        return -result;
    };
}
