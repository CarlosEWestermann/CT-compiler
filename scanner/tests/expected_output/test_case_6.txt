1 TK_PR_BOOL [bool]
1 TK_IDENTIFICADOR [resultado]
1 TK_ESPECIAL [=]
1 TK_IDENTIFICADOR [a]
1 TK_OC_GE [>=]
1 TK_IDENTIFICADOR [b]
1 TK_ESPECIAL [;]
